Static fields and blocks

e4c5 opened the pull request:
How static fields and methods are handled has been improved in a big way. Previously static initializer blocks were not supported and that has been added.

Ishad-M-I-M commented:
![image](https://github.com/user-attachments/assets/63c9f1ab-13d9-4055-9237-09268590d131)
There's a compilation error

e4c5 commented:
Interesting because what i pushed was code that was passing all 232 tests (number of tests have decreased in recent times as i have removed a bunch of tests). But when i try mvn compile you are right indeed it was failing. Now fixed.

Ishad-M-I-M commented:
> Interesting because what i pushed was code that was passing all 232 tests (number of tests have decreased in recent times as i have removed a bunch of tests). But when i try mvn compile you are right indeed it was failing. Now fixed.

Since it's configured to proceed with failing and error test cases, The complete test coverage won't reflect as in local. (There few cases where the test cases are tightly coupled with local file paths)
![image](https://github.com/user-attachments/assets/8b00bda8-0a00-49fd-bb31-1757d67634f4)
